src/nvim/log.h

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <stdbool.h>
 
+#include "nvim/os/os_defs.h"
```

Comment:
this can be removed (we didn't included it, see https://github.com/neovim/neovim/pull/3239#issuecomment-134795962 )